helevents/migrations/0005_auto_20230201_1241.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2 on 2023-02-01 10:41
```

Comment:
Doing this migration is totally fine since it is generated due to changes in Django. If you want, you can rename the migration to something like `0005_increase_user_model_name_field_length.py`, but up to you! :+1: